.github/workflows/ci.yml

```
@@ -180,7 +180,7 @@ jobs:
           AZURE_STORAGE_KEY: ${{ secrets.AZURE_STORAGE_KEY }}
           AZURE_BUCKET_NAME: ${{ secrets.AZURE_BUCKET_NAME }}
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_COMMUNITY }}
-          TELEMETRY_ENDPOINT: oss.edge.df.f5.com:443
+          TELEMETRY_ENDPOINT: ${{ github.ref_type == 'branch' && github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release-') && 'oss-dev.edge.df.f5.com:443' || 'oss.edge.df.f5.com:443' }}
```

```suggestion
          TELEMETRY_ENDPOINT: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release-') && 'oss-dev.edge.df.f5.com:443' || 'oss.edge.df.f5.com:443' }}
```

yeah I'm saying that it can just be like this.

---

after our conversation offline, I have removed the check for `github.ref_type == branch`.